frontend/src/locales/fr.po

```diff
@@ -72,13 +72,13 @@ msgstr "Action"
 
 #: src/dialog/photo/files.vue:29
 msgid "Actions"
-msgstr "Actions"
+msgstr "Images partageant le même identifiant unique d'image."
```

Comment:
It looks like this string doesn't correspond to the original string.

Reply:
Hello Alexis,

Thank you very much for noticing.
I overwrite the translation of this point without realizing it. This is of course unwished.
Technically can I delete the pull request (I did not find how) ? Or just make a corrective change and a new pull request?

Reply:
You don't need to close this PR. You can [merge commits in one commit](https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git/5189600#5189600) then use `push --force` to update the PR.

See also this explanation in French :slightly_smiling_face:  https://www.ekino.com/articles/comment-squasher-efficacement-ses-commits-avec-git

Reply:
Oops, I saw your PR is based on your `develop` branch. It's easier to work on a branch on your fork, then open a PR on this repo to submit your branch. You can create a branch from your `develop` branch, then push it and open a PR. You should see a close button at the bottom of this PR.

Reply:
Thank you very much for your help.
I discover git and the concepts and vocabulary is not easy for me...
I just closed the PR and will do a new one.